api/apis/process_handler.go

```diff
@@ -94,17 +94,7 @@ func (h *ProcessHandler) processGetSidecarsHandler(authInfo authorization.Info,
 
 	_, err := h.processRepo.GetProcess(ctx, authInfo, processGUID)
 	if err != nil {
-		switch err.(type) {
-		case repositories.NotFoundError:
-			h.logger.Info("process not found", "ProcessGUID", processGUID)
-			writeNotFoundErrorResponse(w, "Process")
-		case repositories.ForbiddenError:
-			h.logger.Info("process not accessible to user", "ProcessGUID", processGUID)
-			writeNotFoundErrorResponse(w, "Process")
-		default:
-			h.logger.Error(err, "Failed to fetch process from Kubernetes", "ProcessGUID", processGUID)
-			writeUnknownErrorResponse(w)
-		}
+		h.logError(w, processGUID, err)
```

Comment:
It might be a good opportunity to rename `logError` to something more reflective of what the method is doing. If you don't look at the logError code, you might think it just logs an error and then the handler returns, setting the default StatusOK code. In fact, its main responsibility is writing the appropriate error response.